### PR TITLE
Replace Travis CI with GitHub Actions

### PR DESCRIPTION
**Pull request by tongpu**
##### SUMMARY
Replace Travis CI with GitHub Actions.

##### ISSUE TYPE
 - Feature Pull Request

##### ANSIBLE VERSION
irrelephant

##### ADDITIONAL INFORMATION
Just a first try to trying to replace Travis CI with GitHub Actions.

### CODE REVIEW

File: .github/workflows/ansible-ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: ansible-lint
+      uses: ansible/ansible-lint-action@v4.1.0.post0
+      with:
+        args: "-R -r ../lint-rules"
```
Comment (eyenx) [inline .github/workflows/ansible-ci.yml:20]: we should first git clone our lint rules?

Comment (tongpu) [reply .github/workflows/ansible-ci.yml:20]: Yes, but somehow I was not able to figure out how to do that correctly.

File: .github/workflows/ansible-ci.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    branches: [ master ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
```
Comment (eyenx) [inline .github/workflows/ansible-ci.yml:16]:
```suggestion
    steps:
    - uses: actions/checkout@v2
      with:
        repository:  https://github.com/adfinis-sygroup/ansible-lint-rules
        path: ../lint-rules
```

@tongpu According to the [documentation](https://github.com/actions/checkout)

Comment (tongpu) [reply .github/workflows/ansible-ci.yml:16]: Let's try it.

**Pull request merged**